learningequality/kolibri-design-system
# document release process

## Description






Work on documenting release process for this project:

https://www.notion.so/learningequality/KDS-Release-Policy-is-documented-f5a42b3aa97b47e6ad4a6464e9d89b9b


Also adds a new `DocsShowCode` component with syntax highlight:

https://github.com/learningequality/kolibri-design-system/pull/250/files#diff-48b7f86bd2c62314462f6934da86e5595d4a58a21adf3db1a9019c2c94591f9a

![image](https://user-images.githubusercontent.com/2367265/124962454-a3e67f00-dfec-11eb-9e45-f4c63c1a8641.png)


## Steps to test

See new 'Release process' page:

https://deploy-preview-250--kolibri-design-system.netlify.app/release-process/

## (optional) Implementation notes

### At a high level, how did you implement this?

* Removed a section of 'home'
* Renamed 'Version history' to 'Changelog'
* Migrated work previously done in Notion to KDS
* Added new wrapper component for Prismjs syntax highlighting



## Reviewer guidance




## Comments
